client/components/forms/form-select/index.jsx

```diff
@@ -12,13 +12,15 @@ const FormSelect = React.createClass( {
 	},
 
 	render() {
-		const { className, isError, ...props } = this.props,
-			classes = classNames( className, 'form-select', {
-				'is-error': isError,
-			} );
+		const { inputRef, className, isError, ...props } = this.props;
+		const classes = classNames( className, 'form-select', {
+			'is-error': isError,
+		} );
+
+		const ref = inputRef ? { ref: inputRef } : {};
 
 		return (
-			<select { ...props } className={ classes }>
+			<select { ...props } { ...ref } className={ classes }>
```

Comment:
I'm not aware of a specific problem, but `react-redux` ([withRef](https://github.com/reactjs/react-redux/blob/master/docs/api.md#arguments-1)) and react ([special props](https://facebook.github.io/react/warnings/special-props.html)) both have special handling for the `ref` property, so a defensive stance seemed appropriate to me.

If it bugs you, we could experiment with null refs, but we could always do that later.